ultrasonic/src/main/res/xml/settings.xml

```diff
@@ -343,16 +349,14 @@
             a:summary="@string/settings.hide_media_summary"
             a:title="@string/settings.hide_media_title"
             app:iconSpaceReserved="false"/>
-    </PreferenceCategory>
-    <PreferenceCategory
-        a:title="@string/settings.features_title"
-        app:iconSpaceReserved="false">
```

Comment:
I have removed the one item category "Features" because the star preference can be better grouped somewhere else.